rskj-core/src/main/java/co/rsk/net/handler/TxsPerAccount.java

```diff
@@ -25,6 +25,7 @@
 import java.util.LinkedList;
 import java.util.List;
 
+//TODO: Maybe move this to a test class?
 public class TxsPerAccount {
```

Comment:
Since this class isn’t being used, it might be best to remove the TODO and delete the class